src/models/room.ts

```diff
@@ -3461,16 +3564,23 @@ export class Room extends ReadReceipt<RoomEmittedEvents, RoomEventHandlerMap> {
 
         // get members that are NOT ourselves and are actually in the room.
         let otherNames: string[] = [];
-        if (this.summaryHeroes) {
+        if (this.heroes) {
             // if we have a summary, the member state events should be in the room state
```

Comment:
this comment feels like it's got a bit lost, as well

Reply:
Mmm yeah, should be better now.